services/blockstorage/adapter/filesystem/file_system_persistence.go

```diff
@@ -213,6 +214,7 @@ func newFileBlockWriter(file *os.File, codec blockCodec, nextBlockOffset int64)
 }
 
 func buildIndex(r io.Reader, firstBlockOffset int64, logger log.Logger, c blockCodec) (*blockHeightIndex, error) {
+	r = bufio.NewReaderSize(r, 1024*1024)
```

Comment:
I suggest to create a constant that explains this number.

Reply:
@ronnno Good idea, maybe for a separate PR?

Reply:
Can you please open an issue and assign it to me?

Reply:
issue #1382